script/setup

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Creates the directory where DynamoDbLocal will store its state as a file
+# Creates a table with the the right schema
+
+mkdir -p ~/.gu/security-hq
+
+aws dynamodb create-table --table-name security-hq-iam-TEST \
```

Comment:
shall we name it `-DEV`? to distinguish it from any tables we might create as part of an integration test project?

Reply:
Yes, great point. I forgot about the table name situation. The application picks up the table name from the  `IAM_DYNAMO_TABLE_NAME` in config, which we all currently set to production. I'll add a note to the README

Reply:
What do you think of using just `security-hq-iam`? Given it's the name of the table in prod now and the value of `IAM_DYNAMO_TABLE_NAME`, it would simplify the setup a bit.

Reply:
Is it adding a little bit of risk whereby if someone accidentally sets their stage to `PROD` locally (or there's a coding error) then we're immediately talking to the real table?

Reply:
Oh it totally does, great point!